config/config.go

```diff
@@ -1582,6 +1584,19 @@ func (c *Config) getFieldInt64(tbl *ast.Table, fieldName string, target *int64)
 	}
 }
 
+func (c *Config) getFieldRuneSlice(tbl *ast.Table, fieldName string, target *[]rune) {
+	if node, ok := tbl.Fields[fieldName]; ok {
+		if kv, ok := node.(*ast.KeyValue); ok {
+			str, ok := kv.Value.(*ast.String)
+			if !ok {
+				c.addError(tbl, fmt.Errorf("found unexpected format while parsing %q, expecting string format", fieldName))
+				return
+			}
+			*target = []rune(str.Value)
+		}
+	}
+}
+
```

Comment:
How about handling this as "strings" in the settings and convert to runes in the filter? This way we can directly infer from this code what to expect in the settings...

Reply:
@srebhan Do you mean to remove the above method and just use the below snippets instead?

```GO
c.getFieldString(tbl, "namepass_separator", &f.NamePassSeparators)

...

type Filter struct {
    NameDropSeparators string
...

func (f *Filter) Compile() error {
    .
    f.nameDropFilter, err = filter.Compile(f.NameDrop, []rune(f.NameDropSeparators)...)
    .
}
```

Reply:
have made the above changes, let me know if I misunderstood.